woocommerce/woocommerce-ios
# [Analytics Hub] Enforce internal timezone usage for time ranges descriptions

Part of #8146

Why
==========
The `AnalyticsHubTimeRangeSelection` expects a time zone to be passed as a parameter. This timezone should control every single time range data creation inside the Selection class. But currently, the range description generation is not doing that.

How
==========
Adjust the dedicated date formatters used inside the Analytics Hub to request a timezone parameter to be set, avoiding using the local timezone as a fixed one when generating the range UI descriptions. Also updates the Unit tests to verify the timezone correctness.

How to Test
==========
1. Launch the app and click the `See More button in the My Store charts section.
2. Verify that all date range selection are still working as expected

Update release notes:
